### ISSUE TITLE
How do I run the Python3 program on this

### ISSUE THREAD

**Issue by micocube** [wsdjeg/SpaceVim#1393]
How can I run the Python3 program on this? My system is MacOS, but the system provides Python is 2.7, I open a Python program, press ' space  l r ', use the default python2.7 invoke my program

## Environment Information
- OS:macos high sierra 10.13.1
- neovim version:NVIM v0.2.2

**Comment by wsdjeg** [comment]
do you have `'#!/usr/bin/env python'` on the top?

**Comment by micocube** [comment]
no,just like this:




-------------------------------
赖丁山 18701495141


On 02/9/2018 16:02，Wang Shidong<notifications@github.com> wrote：

do you have '#!/usr/bin/env python' on the top?

**Comment by micocube** [comment]
no,I have changed this line over my python3 path, but it doesn't work.

**Comment by wsdjeg** [comment]
ok, I will fix it.

**Comment by micocube** [comment]
# I added configuration to my ~/spacevim.d/init.vim: 
```
let g:python_host_prog = '/usr/bin/python'
let g:python3_host_prog = '/Library/Frameworks/Python.framework/Versions/3.6/bin/python3'
let g:loaded_python_provider = 1
let g:spacevim_enable_guicolors = 0
" let g:jedi#force_py_version = 3
```
# checkhealth information：
 40 ## Performance
 39   - OK: Build type: Release
 38
 37 ## Remote Plugins
 36   - OK: Up to date
 35
 34 ## terminal
 33   - INFO: key_backspace (kbs) terminfo entry: key_backspace=^H
 32   - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~
 31   - INFO: $XTERM_VERSION=''
 30   - INFO: $VTE_VERSION=''
 29   - INFO: $SSH_TTY=''
 28
 27 health#provider#check
 26 ========================================================================
 25 ## Clipboard (optional)
 24   - OK: Clipboard tool found: pbcopy
 23
 22 ## Python 2 provider (optional)
 21   - INFO: Disabled. g:loaded_python_provider=1
 20
 19 ## Python 3 provider (optional)
 18   - INFO: Using: g:python3_host_prog = "/Library/Frameworks/Python.framework/Versions/3.6/bin/python3"
 17   - INFO: Executable: /Library/Frameworks/Python.framework/Versions/3.6/bin/python3
 16   - INFO: Python3 version: 3.6.1
 15   - INFO: python3-neovim version: 0.2.1
 14   - OK: Latest python3-neovim is installed: 0.2.1
 13
 12 ## Ruby provider (optional)
 11   - INFO: Ruby: ruby 2.3.3p222 (2016-11-21 revision 56859) [universal.x86_64-darwin17]
 10   - WARNING: Missing "neovim" gem.
  9     - ADVICE:
  8       - Run in shell: gem install neovim
  7       - Is the gem bin directory in $PATH? Check `gem environment`.
  6       - If you are using rvm/rbenv/chruby, try "rehashing".
  5
  4 ## Node provider (optional)
  3   - WARNING: `node` and `npm` must be in $PATH.
  2     - ADVICE:
  1       - Install Node.js and verify that `node` and `npm` commands work.

**Comment by wsdjeg** [comment]
this  should be fixed in #1394
